app/components/filters_component.html.erb

```diff
@@ -9,7 +9,11 @@
             'action': 'filters#toggleDisplayFilters',
             'test-selector': 'filter-component-toggle' })) do %>
       <%= t(:label_filter) %>
-      <%= render(Primer::Beta::Counter.new(count: filters_count, round: true, hide_if_zero: false, scheme: :default)) %>
+      <%= render(Primer::Beta::Counter.new(count: filters_count,
+                                           round: true,
+                                           hide_if_zero: false,
+                                           scheme: :default,
+                                           data: { "test-selector": "filters-button-counter" })) %>
```

Comment:
```suggestion
                                           test_selector: "filters-button-counter" )) %>
```

The system_arguments have a test_selector that allows you to use without having to use `data: { ..}`